setup.cfg

```diff
@@ -4,7 +4,7 @@ universal = 1
 
 [metadata]
 requires-dist =
-        botocore==2.0.0dev36
```

Comment:
At time of this commit, botocore `2.0.0dev36` doesn't even exist, and the HEAD of the v2 branch of https://github.com/boto/botocore is `2.0.0dev35`.